src/ddmd/expression.d

```diff
@@ -515,6 +526,8 @@ extern (C++) Expression resolveProperties(Scope* sc, Expression e)
 
 /******************************
  * Check the tail CallExp is really property function call.
+ * Bugs:
+ * This doesn't appear to do anything.
```

Comment:
Because this PR is about adding documentation. It's good practice to avoid "mission creep" in PRs.